admin-frontend/open_admin_app/lib/widgets/user/signin/signin_widget.dart

```diff
@@ -109,7 +109,8 @@ class _SigninState extends State<SigninWidget> {
               padding: const EdgeInsets.only(bottom: 26.0),
               child: Row(
                 mainAxisAlignment: MainAxisAlignment.center,
-                children: const <Widget>[RotatingLogoWidget()],
```

Comment:
this is also used in the register page, should we remove it and remove the whole widget?

Reply:
this is register page, it's not used anywhere else. I'd like to keep the widget itself, might use in the future somewhere else.